setup.py

```diff
@@ -30,6 +30,7 @@
         'cfscrape>=2.0.5',
         'requests-cache>=0.4.13',
         'tabulate>=0.8.3',
+        'PyQt5>=5.15.1',
```

Comment:
Pyqt doesn't install on android (at least via termux) - so this breaks anime-downloader on android, and it's only needed by people who will use the GUI - perhaps move this to `extras_require`?

Reply:
That's a great idea. I'll do that